README.org

```diff
@@ -4,6 +4,14 @@
 
 Convert =go.mod= files to nixpkgs =buildGoPackage= compatible =deps.nix= files
 
+** Installation
+
+You can install the most recent vgo2nix by executing
+
+#+begin_src
+nix-env -i vgo2nix-git -f https://github.com/adisbladis/vgo2nix/archive/master.tar.gz
```

Comment:
Since we have it in nixpkgs, this should be thee preferred way of installing really.

Reply:
Do you suggest something like `nix-env -i vgo2nix -f https://github.com/nixos/nixpkgs/archive/9875bbae75231f395ab020fbef6cbae698cc75c5.tar.gz` @Mic92?

Reply:
I think you can do the installation instructions similar to this: https://github.com/Mic92/nixpkgs-review#installation

Reply:
Do you mean `nix run nixpkgs.vgo2nix`?

Reply:
I mean it could lists several methods on how to run vgo2nix from nixpkgs as well as how to build and run it from the repository.